microsoft/PowerToys
# [KBM] Modifier Key (Not right or left) stuck problem solved.

## Summary of the Pull Request
When shortcut set a modifier key without left or right, It sticks. To solve that if modifier is "both", event is set with which actual modifier key is pressed.

!!! Please first merge https://github.com/microsoft/PowerToys/pull/32125 and then merge main to this branch and wait for any code conflicts to be resolved before merging this branch.

## PR Checklist

- [ ] **Closes:** #17035 


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed

- Set shortcut Ctrl + any key
- Press Ctrl (Left) + key
- Press Ctrl (Right) + key
- Check if Ctrl key is stuck or not
